.github/workflows/danger.yml

```diff
@@ -0,0 +1,21 @@
+name: Danger
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  danger:
+    runs-on: macos-latest
```

Comment:
why macos? it's expensive

Reply:
It’s free in public repos. But I can switch to Ubuntu